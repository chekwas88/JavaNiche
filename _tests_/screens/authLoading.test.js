import React from 'react';
import { shallow } from 'enzyme';
import AuthScreen from '../../src/screens/AuthLoading';

describe('Test for Login component', () => {
  const wrapper = shallow(<AuthScreen />);
  test('should render authLoading correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
