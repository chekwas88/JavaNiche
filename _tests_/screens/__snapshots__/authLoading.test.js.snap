// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for Login component should render authLoading correctly 1`] = `ShallowWrapper {}`;
